.travis.yml

```diff
@@ -3,7 +3,7 @@ language: go
 go_import_path: go.uber.org/yarpc
 cache:
   directories:
-    - $HOME/.yarpc-go
+    - $HOME/.cache/yarpc-go
```

Comment:
I effectively assumed that travis doesn't set `XDG_CACHE_HOME`. In fact XDG_CACHE_HOME is very rarely set on most distros. This is because the default "~/.cache" is perfectly reasonable. On my linux system, only non-standards settings are set via `XDG_*` env vars for example (mostly i3 config and data dirs).

But you raise a very valid point, I will use some env var.